cmd/client/lib/cli.go

```diff
@@ -160,35 +162,38 @@ func Create(c *cli.Context, withInstrumentation bool, opts ...client.Option) (cl
 }
 
 func buildGrpcClient(c *cli.Context, info **chain.Info) ([]client.Client, error) {
-	if c.IsSet(GRPCConnectFlag.Name) {
-		hash := make([]byte, 0)
+	if !c.IsSet(GRPCConnectFlag.Name) {
+		return []client.Client{}, nil
```

Comment:
I like this change. Wondering if this shouldn't be nil. Just having you double check this.